### ISSUE TITLE
Retrieving OCR'able images from Discord

### ISSUE THREAD

**Issue by HeliumxU** [RogueException/Discord.Net#1284]
public string Url { get; } - this is how you get the image from discord afaik, but there is no way to incorporate this into another string, f.example a OCR? I can't see how, because "url { get; }" using context does not produce any workable results

If anyone could help me it'd be fantastic, because it'd improve translating tenfold, by letting users directly grab the text from an image and then chucking it into WeRead or Google Translate etc.

``
        {
            AutoOcr OCR = new AutoOcr() { ReadBarCodes = false };
            var Results = OCR.Read("*URL FOR IMAGE*");
            await Context.Channel.SendMessageAsync(Results.Text + "");
        }
``

**Comment by Still34** [comment]
This issue does not directly relate to Discord.Net and thus is not a valid issue; consider using other resources such as [StackOverflow](https://stackoverflow.com).

**Comment by HeliumxU** [comment]
Okay, so get url not outputting anything is not a discord.net issue? I thought that was Discord.Net? or is it something else like the http client?

**Comment by Still34** [comment]
Perhaps I'm missing something here; could you rephrase your issue?

**Comment by HeliumxU** [comment]
Okay, issue is like this. Discord.Net issues I've checked have said you can do "get url" and save images to the computer via that method, but it doesn't download, and returns a "null" and gets caught as an exception.

**Comment by Still34** [comment]
The `get;` refers to retrieving the URL. You need a client like HttpClient to download the file. Furthermore, this still isn't classified as a valid issue, as it still does not seem to be a critical library issue. Please visit us on the Discord API server for general questions.
